lib/Contracts/IMailManager.php

```diff
@@ -317,6 +317,18 @@ public function createTag(string $displayName, string $color, string $userId): T
 	 */
 	public function updateTag(int $id, string $displayName, string $color, string $userId): Tag;
 
+	/**
+	 * Delete a mail tag
+	 *
+	 * @param int $id
+	 * @param string $userId
+	 * @param Account $account
+	 *
+	 * @return Tag
```

Comment:
```suggestion
```

The phpdoc for param & return is only duplicates the method signature but does not provide additional information and therefore should go. 

Please keep the method description and throws.